packages/plugin-global-user-properties/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@amplitude/plugin-global-user-properties",
+  "version": "0.1.0",
+  "description": "",
+  "author": "Amplitude Inc",
+  "homepage": "https://github.com/amplitude/Amplitude-TypeScript",
+  "license": "MIT",
+  "main": "lib/cjs/index.js",
+  "module": "lib/esm/index.js",
+  "types": "lib/esm/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public",
+    "tag": "latest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/amplitude/Amplitude-TypeScript.git"
+  },
+  "scripts": {
```

Comment:
Do we want to upload this to cdn? if yes, add `"publish": "node ../../scripts/publish/upload-to-s3.js"` will upload the script into S3.

Reply:
we do not need to publish to cdn, since this will be used on sites with npm, thanks for asking about this though!